Drop-In/src/main/java/com/braintreepayments/api/CardDetailsFragment.java

```diff
@@ -195,4 +200,15 @@ public void onCardFormFieldFocused(View view) {
             sendDropInEvent(DropInEvent.createEditCardNumberEvent(cardNumber));
         }
     }
+
+    private void hideSoftKeyboard() {
+        // Ref: https://stackoverflow.com/a/3553811
+        FragmentActivity activity = getActivity();
+        if (activity != null) {
+            InputMethodManager imm =
```

Comment:
nit: take it or leave it but it may be helpful to be more explicit in the naming here vs `imm`